fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/bottomsheet/BottomSheet.kt

```diff
@@ -514,13 +555,32 @@ private fun Modifier.bottomSheetSwipeable(
                 fullHeight - peekHeightPx to BottomSheetValue.Shown
             )
         }
-        Modifier.swipeable(
-            state = sheetState,
-            anchors = anchors,
-            orientation = Orientation.Vertical,
-            enabled = sheetState.currentValue != BottomSheetValue.Hidden,
-            resistance = null
-        )
+        if (stickyBehavior) {
+            Modifier.swipeable(
+                state = sheetState,
+                anchors = anchors,
+                orientation = Orientation.Vertical,
+                enabled = sheetState.currentValue != BottomSheetValue.Hidden,
+                thresholds = { from, to ->
+                    val fromKey = anchors.entries.firstOrNull { it.value == from }?.key
+                    val toKey = anchors.entries.firstOrNull { it.value == to }?.key
+                    if (fromKey != null && toKey != null && fromKey < toKey) {
```

Comment:
Should we have a null check outside of this if condition. Looks like line 570 is executed even when from ot to keys are null. Please check if this is intended.

Reply:
Now, first null check is done then other check